File: DBtoCSharp_MARIADB/output/INNODB_TABLESPACES_ENCRYPTION.ts
export interface INNODB_TABLESPACES_ENCRYPTION
    {
        /**///  */
        SPACE: number;
        /**///  */
        NAME: string;
        /**///  */
        ENCRYPTION_SCHEME: number;
        /**///  */
        KEYSERVER_REQUESTS: number;
        /**///  */
        MIN_KEY_VERSION: number;
        /**///  */
        CURRENT_KEY_VERSION: number;
        /**///  */
        KEY_ROTATION_PAGE_NUMBER: number;
        /**///  */
        KEY_ROTATION_MAX_PAGE_NUMBER: number;
        /**///  */
        CURRENT_KEY_ID: number;
        /**///  */
        ROTATING_OR_FLUSHING: number;
}

File: DBtoCSharp_MARIADB/modelos/INNODB_TABLESPACES_ENCRYPTION.cs

using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Models
{
    public class INNODB_TABLESPACES_ENCRYPTION
    {
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public int SPACE { get; set; }
            [StringLength(655)]
        /// <summary>
        /// 
        /// </summary>
        public string NAME { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public int ENCRYPTION_SCHEME { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public int KEYSERVER_REQUESTS { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public int MIN_KEY_VERSION { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public int CURRENT_KEY_VERSION { get; set; }
        
        /// <summary>
        /// 
        /// </summary>
        public long KEY_ROTATION_PAGE_NUMBER { get; set; }
        
        /// <summary>
        /// 
        /// </summary>
        public long KEY_ROTATION_MAX_PAGE_NUMBER { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public int CURRENT_KEY_ID { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public int ROTATING_OR_FLUSHING { get; set; }

}
}
